scripts/download_re2.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+args=("$@")
+re2_version=${args[0]}
+
+if [ -z "$re2_version" ]; then
+  echo "Usage: ./download_re2.sh [re2_version]"
+  exit 1
+fi
+
+archs=(
+  "darwin-arm64"
+  "darwin-x64"
+  "linux-musl-x64"
+  "linux-x64"
+  "win32-x64"
+)
+node_api_versions=( 108 115 )
+formats=( "br" "gz" )
```

Comment:
What is the `br` file used for? Wondering if we are using both.

Reply:
I was thinking the same thing. Both existed in bucket already, so I just did the same thing this time around. But I actually don't think we need it. Should I remove it in this PR or do y'all prefer that we do that at a later stage? I think there's a bunch of stuff regarding how we store these files that probably needs to be "cleaned up" now that we also have the custom build of Node.js and re2 in another bucket as well

Reply:
I did a quick search for anything using `br` and didn't find anything. Maybe best to leave it for now and do a more thorough check in the clean up pass.